src/ipc-handlers.ts

```diff
@@ -712,7 +715,7 @@ export async function archiveSite( event: IpcMainInvokeEvent, id: string, format
 	return { archivePath, archiveSizeInBytes: stats.size };
 }
 
-export async function exportSiteToPush(
+export async function exportSiteForPush(
```

Comment:
Renaming this function is also a secondary function of this PR. I did it because I think this clarifies the relationship better between this function and `pushArchive`.